src/resources/common.yml

```diff
@@ -405,3 +405,10 @@ deletedRoom:
 deletedLesson:
   en: Deleted lesson
   de: Gelöschte Einheit
+reset:
+  en: Reset
+  de: Zurücksetzen
+randomizedTests:
+  en: The tests are randomised
+  de: Die Tests sind randomisiert
```

Comment:
Sounds too academic in German. We have to say smth like: "Die Fragen sind in einer zufälligen Reihenfolge angeordnet" (the questions are arranged in random order)